fsharp-backend/src/LibExecution/StdLib/LibInt.fs

```diff
@@ -107,13 +107,13 @@ let fns : List<BuiltInFn> =
         (function
           | _, [ DInt a; DInt b ] -> Value(DInt(a + b))
           | _, [ (DFloat _ as a); _ ] ->
-            Value(errStr("The first param" + a.ToString() + "is a float, but only works on Ints. Use Float::add to compare Floats or use Float::truncate to truncate Floats to Ints."))
+            Value(errStr($"The first argument ({a}) is a float, but Int:add only works on Ints. Use Float::add to compare Floats or use Float::truncate to truncate Floats to Ints."))
           | _, [ _; DFloat _ as a ] ->
-            Value(errStr("The second param" + a.ToString() + "is a float, but only works on Ints. Use Float::add to compare Floats or use Float::truncate to truncate Floats to Ints."))
+            Value(errStr($"The second argument ({a}) is a float, but Int:add only works on Ints. Use Float::add to compare Floats or use Float::truncate to truncate Floats to Ints."))
           | _, [ DStr a; _ ] ->
-            Value(errStr("The first param" + a.ToString() + "is a string, but only works on Ints."))
+            Value(errStr($"The first argument ({a}) is a string, but Int:add only works on Ints."))
           | _, [ _; DStr _ as a ] ->
```

Comment:
Can you switch to `DStr a` rather than `DStr _ as a`? Otherwise the error message will say `DStr "string"` in it.